src/component/home/Home.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from './Home.module.css';
+
+const Home = () => (
+  <div className={styles.home}>
+    <h2>
+      Welcome to our page
+    </h2>
+    <p>
+      Lorem ipsum dolor sit amet consectetur
+      adipisicing elit. Velit, quis explicabo. Quam amet quo blanditiis dolor?
+      Quod voluptatum eum velit expedita repellat laboriosam sunt blanditiis optio,
+      nihil culpa neque quos laborum corporis dolorum, tempore ipsam.
+      Tempore eius nulla explicabo harum!
+
+    </p>
+    <p>
+      Lorem ipsum dolor sit amet consectetur
+      adipisicing elit. Velit, quis explicabo. Quam amet quo blanditiis dolor?
+      Quod voluptatum eum velit expedita repellat laboriosam sunt blanditiis optio,
+      nihil culpa neque quos laborum corporis dolorum, tempore ipsam.
+      Tempore eius nulla explicabo harum!
```

Comment:
- [ ] Kindly define what your calculator application other than using `Lorem`. This will make your application look more professional and stand out to users.